src/featureFlags/flags/featureFlags.ts

```diff
@@ -1,6 +1,5 @@
 export enum FeatureFlag {
-  favoriteTokens = 'favoriteTokens',
```

Comment:
is there a lint rule for unused enum variants lol

Reply:
there should be, but I can't find one